### ISSUE TITLE
Dropdown: screen reader does not announce 'expanded' when expanded

### ISSUE THREAD

**Issue by natalieethell** [OfficeDev/office-ui-fabric-react#9374]
**Browser + screen reader combo:** Edge + Narrator

Narrator does not read out the 'expanded' state when the dropdown is expanded. I think this is happening, because the focus goes to the selected option when opening a dropdown, not the dropdown itself, which has the 'expanded/collapsed' state.

## Codepen demonstrating the issue
https://codepen.io/naethell/pen/EzBmwq

## Actual behavior:
When focused on a collapsed dropdown control, Narrator announces 'collapsed', but when expanding the control by hitting the space bar or 'Enter' key, it does not announce 'expanded'. 

## Expected behavior:
Narrator should announce 'expanded' when expanding the dropdown control.

**Comment by khmakoto** [comment]
@natalieethell like you said, this is happening because the focus goes automatically to the selected option when opening the `Dropdown`. We could certainly hack our way around this, but since this is expected behavior of the screen reader given what's happening I'd rather not do anything here.

**Comment by xugao** [comment]
Agree with Mak here. I'm going to close this one

**Comment by xugao** [comment]
Here is WI for Edge to not hide this as advanced info: https://microsoft.visualstudio.com/OS/_workitems/edit/25287194
